ocean-client/src/utils/store_aws_reinvest.ts

```diff
@@ -55,6 +56,7 @@ export class StoreAWSReinvest extends StoreAWS implements IStore {
     let ReinvestThreshold = this.postfixedKey(StoreKey.ReinvestThreshold)
     let AutoDonationPercentOfReinvestKey = this.postfixedKey(StoreKey.AutoDonationPercentOfReinvest)
     let LMPairKey = this.postfixedKey(StoreKey.LMPair)
```

Comment:
LMPairKey can be removed, right?

Reply:
keeping it for backward compatibility (ppl updating manually)